datadog/resource_datadog_integration_pagerduty_test.go

```diff
@@ -149,19 +148,8 @@ func datadogIntegrationPagerdutyExistsHelper(client *datadog.Client) error {
 
 func testAccCheckDatadogIntegrationPagerdutyDestroy(accProvider *schema.Provider) func(*terraform.State) error {
```

Comment:
Could we just remove this method and the CheckDestroy that uses them?

Reply:
It's related to https://github.com/terraform-providers/terraform-provider-datadog/pull/584/files#diff-f2ab83c1cfbd1f73e2279cdc962210a0L209-L217

I missed that part.

Reply:
It seems I need to keep it, it fails weirdly without it.